docs/reference/modules/discovery/zen.asciidoc

```diff
@@ -82,6 +82,12 @@ serves as a protection against (partial) network failures where node may unjustl
 think that the master has failed. In this case the node will simply hear from
 other nodes about the currently active master.
 
+If the setting `discovery.zen.master_election.filter_client` is `true`, pings from client nodes (or non-data,
+non-master-eligible nodes) will be ignored during master election; the default value is `true`. If the setting
```

Comment:
Looks good! A few very minor editorial comments you can do what you will with:

- You can drop "the setting", just say "If xyz is true..."
- Stick with present tense when possible--instead of "pings...will be ignored", just say "pings...are ignored". 
- Maybe "acknowledged" instead of "observed"?

I might combine your approach and Clint's suggestion into:

"pings from client nodes (nodes where node.client is set to true and node.data and node.master are false)"
"pings from data nodes (nodes where node.data is set to true and node.master is false)"

Your call. I just think it's a little easier to parse "client node" and "data node" than "nodes with xyz settings".

FYI, Haley has been assembling a style guide that we're trying to sync on for both marketing info and the docs. It's at https://elasticsearch.atlassian.net/wiki/display/MAR/Writing+Style+Guide.